### DISCUSSION TITLE
[Feature] Make mobile apps easily findable by typing something like Photos

### DISCUSSION THREAD

**Discussion by piyushpaliwal** [immich-app/immich#14161] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

My entire family uses immich to store their photos and share them across. However, one thing I have constantly heard from them is

"Hey, what was the name of that app again?"

Now I know, this might sound silly but this is a genuine problem that my dad, mom and elder sibling face. So, I was thinking if there is a way to show up Immich in the search results when someone types "Photos" or something similar.

The previous apps they have used are Google Photos, and Photos app on the iPhone. 

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by alextran1502** [comment]
I think the search uses the app name. It doesn’t use the app metadata afaik

**Reply by crazicus** [reply]
I don’t know if Android supports this, but iOS does. It seems to pull from the name of the App Store Connect listing. In my testing, “Bank” reveals Ally and Chase, “Taxi” yields Uber as a result, “Podcast” returns Spotify, etc.
